### ISSUE TITLE
First child of <fieldset> should be <legend> (WCAG compliance)

### ISSUE THREAD

**Issue by ekim-novus** [surveyjs/surveyjs#1445]
### Are you requesting a feature, reporting a bug or asking a question?

Requesting a feature (although sort of a bug as well?)

### What is the current behavior?

For radiogroups, checkbox groups, and question matrices (and anywhere else that uses <fieldset>, the first child of the <fieldset> tag is not a <legend> tag, as required by W3C:
https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140408/H71

The legend tag is also necessary for WCAG compliance.

### What is the expected behavior?

As shown below in **Proposed code**, I would suggest:
1) moving the <legend> to first child of <fieldset>
2) removing `display: none` from <legend>
3) removing the innerHTML from <legend>, and setting it equal to `aria-label` instead, so that it is visually hidden but will be recognized by screen readers

#### Provide the test code and the tested page URL (if applicable)

Proposed code (example for reactquestionradiogroup.tsx)

```
<fieldset className={cssClasses.root}>
        <legend aria-label={this.question.locTitle.renderedHtml}>
        </legend>
        {this.getItems(cssClasses)}
        {clearButton}
</fieldset>

```

### Specify your

* browser: Chrome
* browser version: 70.0.3538.77
* surveyjs platform (angular or react or jquery or knockout or vue): react
* surveyjs version: 1.0.56

**Comment by tsv2013** [comment]
Thanks for the suggestion. I've implemented it.

**Comment by dszewczyk1** [comment]
It seems that <legend> is only added to <fieldset> with checkboxes, but not for radio group?
